src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -311,6 +311,33 @@ public GuildImpl createGuild(long guildId, DataObject guildJson, TLongObjectMap<
         return guildObj;
     }
 
+    public Channel createChannel(DataObject dataObject){
```

Comment:
Since we are not going to expose the method publicly, we can remove this for now.